packages/terra-tabs/src/common-tabs/_Tab.jsx

```diff
@@ -230,6 +230,10 @@ const Tab = ({
         onChange(event, itemKey);
       }
     }
+    // Fix for keyboard navigation after mouse click which was failing due to draggable props.
+    if (isDraggable) {
+      event.currentTarget.focus();
```

Comment:
so now the focus always remains on the dragged tab ? and once dropped the keyboard navigation starts from that position ?

Reply:
Before the change when a draggable tab was clicked, the focus was going onto the main or the body div due to the draggable props of the dnd library. So I have put the focus on the clicked tab. Change has nothing to do with whether the tab has been dragged or not.